content/en/serverless/installation/java.md

```diff
@@ -31,9 +31,68 @@ To fully instrument your serverless application with distributed tracing, your J
 If your Java Lambda functions were previously set up using the Datadog Forwarder, see the [installation instructions][1].
 
 ## Configuration
+
+{{< tabs >}}
+{{% tab "Datadog CLI" %}}
+The Datadog CLI modifies existing Lambda functions' configurations to enable instrumentation without requiring a new deployment. It is the quickest way to get started with Datadog's serverless monitoring.
+
+You can also add the command to your CI/CD pipelines to enable instrumentation for all your serverless applications. Run the command _after_ your normal serverless application deployment, so that changes made by the Datadog CLI command are not overridden.
```

Comment:
```suggestion
You can also add the [instrumentation command](#instrumentation) to your CI/CD pipelines to enable instrumentation for all your serverless applications. Run the command _after_ your normal serverless application deployment, so that changes made by the Datadog CLI command are not overridden.
```